docs/command_line_reference.rst

```diff
@@ -510,6 +510,8 @@ Example::
    # Force to run with JDK 7
    esrally --distribution-version=2.4.0 --runtime-jdk=7
 
+There is also limited support to specify the JDK that is bundled with Elasticsearch with the special value ``bundled``. At the moment this only works on Linux. The JDK is bundled from Elasticsearch 7.0.0 onwards.
```

Comment:
Does it make sense to link: https://www.elastic.co/blog/elasticsearch-7-0-0-released re: `The JDK is bundled from Elasticsearch 7.0.0 onwards.`?

Reply:
I'd consider a reference documentation more stable than a blog and would thus rather avoid pointing to a blog post?

Reply:
Maybe this one then? https://www.elastic.co/guide/en/elasticsearch/reference/current/release-highlights-7.0.0.html#_bundle_jdk_in_elasticsearch_distribution

Reply:
Addessed in https://github.com/elastic/rally/pull/853/commits/66624207845b8b00a0de4de4c1ebc8ccfd03ddbc

Reply:
Thanks!